pkg/kubelet/dockertools/manager.go

```diff
@@ -231,6 +237,7 @@ func NewDockerManager(
 		cpuCFSQuota:            cpuCFSQuota,
 		enableCustomMetrics:    enableCustomMetrics,
 		configureHairpinMode:   hairpinMode,
+		versionCache:           cache.NewVersionCache(),
```

Comment:
Sorry @vishh , I'm still not quite get your meaning here. In previous comment, I can see `dm.getVersionInfo()` is not a updater method, and it may also cause data race. So would you mind to clarify what you'd like me to refactor here and what duplicated/problem it aims to solve? :)